File: Source/Rio.Web/src/app/shared/models/water-type-dto.ts
export class WaterTypeDto {
    WaterTypeID: number;
    WaterTypeName: string;
    IsAppliedProportionally: WaterTypeApplicationTypeEnum;
    WaterTypeDefinition: string;
    SortOrder: number;

    constructor(obj?: any){
        Object.assign(this, obj);
    }
}

export enum WaterTypeApplicationTypeEnum{
    Spreadsheet = 0,
    Proportional = 1,
    Api = 2
}

File: Source/Rio.Models/DataTransferObjects/WaterTypeDto.cs
namespace Rio.Models.DataTransferObjects
{
    public class WaterTypeDto
    {
        public int WaterTypeID { get; set; }
        public string WaterTypeName { get; set; }
        public WaterTypeApplicationTypeEnum IsAppliedProportionally { get; set; }
        public string WaterTypeDefinition { get; set; }
        public int SortOrder { get; set; }
    }

    public enum WaterTypeApplicationTypeEnum
    {
        Spreadsheet = 0,
        Proportionally = 1,
        Api = 2
    }
}
